src/pkg/skopeo/skopeo.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2019 – 2022 Red Hat Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package skopeo
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/containers/toolbox/pkg/shell"
+)
+
+func Inspect(typearg string, target string) (interface{}, error) {
```

Comment:
The `typearg` argument is unused, so it might be better to not have it.

The reason `podman.Inspect` has a `typearg` argument is because one can use `podman inspect` with both OCI containers and images.  However, that's not the case with `skopeo inspect` because it can only be used with images.